lib/aurora_bootstrapper/notifier.rb

```diff
@@ -26,6 +67,10 @@ def notify
     end
     
     protected
+
+    def convert_date_to_string( date: )
+      date.strftime("%Y-%m-%d")
+    end
```

Comment:
do we need this function, or should we have a constant `DATE_FORMAT` that we can feed into `strftime`?

Reply:
Removed it